views/js/previewer/plugins/content/scoringHighlighter.js

```diff
@@ -106,24 +130,42 @@ define([
 
                 //Sending the highlighIndex to parent so that it can be saved on MS side
                 selection.removeAllRanges();
-                window.parent.postMessage({ event: 'indexUpdated', payload: highlighter.getHighlightIndex() }, '*');
+                const highlightIndex = highlighter.getHighlightIndex();
+                window.parent.postMessage({ event: 'indexUpdated', payload: highlightIndex }, '*');
+                this.updateHashighlights(highlightIndex);
```

Comment:
Since that part of code is repeating, could we move it to the independent function?
```javascript
           this.saveHighlightsStatus = () => {
                const highlightIndex = highlighter.getHighlightIndex();
                window.parent.postMessage({ event: 'indexUpdated', payload: highlightIndex }, '*');
                this.updateHashighlights(highlightIndex);
            }
```

Reply:
I agree!